github-adapter/src/main/java/be/xplore/githubmetrics/githubadapter/config/auth/GithubAuthTokenInterceptor.java

```diff
@@ -20,20 +19,24 @@ public class GithubAuthTokenInterceptor implements ClientHttpRequestInterceptor
 
     private static final Logger LOGGER = LoggerFactory.getLogger(GithubAuthTokenInterceptor.class);
     private final GithubProperties githubProperties;
-    private final RestClient tokenFetcherRestClient;
+    private final GithubRestClient githubAuthRestClient;
     private ZonedDateTime tokenExpirationDateTime = now();
     private String currentAccessToken;
 
     public GithubAuthTokenInterceptor(
             GithubProperties githubProperties,
-            @Qualifier("tokenFetcherRestClient") RestClient tokenFetcherRestClient
+            @Qualifier("githubAuthRestClient") GithubRestClient githubAuthRestClient
```

Comment:
is the qualifier still needed?

Reply:
yes as they both still return the same Class of object